src/pages/Signup/index.js

```diff
@@ -47,8 +48,30 @@ const Signup = () => {
     handleSubmit,
     formState: { errors },
   } = useForm({ resolver: zodResolver(schema) });
+  const [isSend, setIsSend] = useState(true);
 
-  const onSubmit = data => signup(data);
+  const [toSend, setToSend] = useState({
+    to_name: '',
+    message: `Hello, thank you for signing up to MVD! We're excited to have you onboard and will be happy to help you set everything up.`,
+    to: '',
+  });
+
+  const sendEmailValidation = data => {
+    setToSend((toSend.to_name = data.username), (toSend.to = data.email));
```

Comment:
You shouldn't mutate `toSend` directly, instead you should do it with `setToSend` passing a new object with the updated values. If you need to, you can access the previous state inside of `setToSend` (https://reactjs.org/docs/hooks-reference.html#functional-updates)

```suggestion
    setToSend((prevState) => {
       return {
          ...prevState,
          to_name: data.username,
          to: data.email
       }
    });
```

Reply:
I will fix this now. Thanks.